tests/imodels-clients-tests/src/unit/management/AxiosRestClient.test.ts

```diff
@@ -2,18 +2,34 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import { AxiosRestClient, IModelsErrorParser } from "@itwin/imodels-client-management/lib/base/internal";
+import { AxiosRestClient, IModelsApiError, AxiosRetryPolicy, IModelsErrorParser } from "@itwin/imodels-client-management/lib/base/internal";
 import axios from "axios";
 import MockAdapter from "axios-mock-adapter";
 import { expect } from "chai";
+import * as utilityFunctions from "@itwin/imodels-client-management/lib/base/internal/UtilityFunctions";
 
 import { ContentType, HttpRequestWithJsonBodyParams } from "@itwin/imodels-client-management";
+import { createStub } from "../Stubs";
+import sinon from "sinon";
 
 describe("[Management] AxiosRestClient", () => {
   let axiosMock: MockAdapter;
+  let retryPolicyStub: sinon.SinonStubbedInstance<AxiosRetryPolicy>;
+  let sleepStub: sinon.SinonStub;
 
   before(() => {
     axiosMock = new MockAdapter(axios);
+    retryPolicyStub = createStub(AxiosRetryPolicy);
+    sleepStub = sinon.stub(utilityFunctions, "sleep");
+  });
+
+  afterEach(() => {
+    axiosMock.resetHandlers();
```

Comment:
I think you can just call `reset`
![image](https://github.com/iTwin/imodels-clients/assets/70565417/64a572ab-ca6b-4143-92b5-c45252fb9152)
https://sinonjs.org/releases/latest/stubs/

Reply:
updated